build.gradle

```diff
@@ -57,13 +57,10 @@ dependencies {
     implementation 'org.ow2.asm:asm-tree:8.0'
     implementation 'org.ow2.asm:asm-util:8.0'
     implementation 'net.sf.jopt-simple:jopt-simple:6.0-alpha-3'
-    implementation 'net.fabricmc:procyon-fabric-compilertools:0.5.35.+'
+    implementation 'net.fabricmc:procyon-fabric-compilertools:0.5.35.13'
     implementation 'net.fabricmc:cfr:0.0.1'
     implementation name: "darcula", version: "1.0.0"
-    implementation 'de.sciss:syntaxpane:1.2.+'
-    implementation 'me.xdrop:fuzzywuzzy:1.2.0'
-    implementation name: "darcula", version: "1.0.0"
```

Comment:
dont we need these 2?

Reply:
They were duplicated in the buildscript.

Reply:
a yeah.